desktop-app/app/menu.js

```diff
@@ -340,6 +342,19 @@ export default class MenuBuilder {
             this.mainWindow.toggleDevTools();
           },
         },
+        {
+          label: 'Show Status Bar',
+          type: 'checkbox',
+          checked: statusBarSettings.getVisibility(),
+          click: () => {
+            const newVisibility = !statusBarSettings.getVisibility();
+            statusBarSettings.setVisibility(newVisibility);
```

Comment:
@falecci Can you please move the `set` logic of the settings to the reducer? 
So that the settings are consistent with the `store` values.

Reply:
Nice catch, you are right! The `set` logic shouldn't be here :). I'll place it on the action creator instead of the reducer, since it makes more sense to have it there.

Reply:
I've renamed the action to `toggleStatusBarVisibility`, so we all the set logic it's encapsulated :)